### DISCUSSION TITLE
using Kinect v2 for in rtabmap-ros ros2

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1191] [General]
HI

I'm using kinect v1, in ros2 everythings fine
now I want to use kinect v2, better quality, lower noise, wider Field of view

how can i use it ?
thank you

**Comment by matlabbe** [comment]
It seems there is a port of the original [ros1 package](https://github.com/code-iai/iai_kinect2) for ros2 here: https://github.com/YuLiHN/kinect2_ros2

If topics have same name, follow instructions about `kinect2_bridge` on this [page](http://wiki.ros.org/rtabmap_ros/Tutorials/HandHeldMapping). See this page to convert those examples in ROS2: https://github.com/introlab/rtabmap_ros/tree/ros2#usage
